src/storage/src/source/source_reader_pipeline.rs

```diff
@@ -1035,21 +1028,16 @@ where
                         tokio::time::sleep(wait_time).await;
                     }
                 } => {
-                    let remap_trace_updates = timestamper.mint(&global_source_upper).await;
```

Comment:
there's already other things called `batch` in this file, maybe keep sth like `remap_trace_batch`?

Reply:
will do